tests/cpplatest/CMakeLists.txt

```diff
@@ -12,11 +12,6 @@ if (${CMAKE_CXX_COMPILER_ID} STREQUAL "Clang")
     target_compile_definitions(witest.cpplatest PRIVATE _SILENCE_CLANG_COROUTINE_MESSAGE)
 endif()
 
-# Semi-arbitrary insiders SDK version selected that uses C++/WinRT "2.0"

```

Comment:
No longer necessary now that we're manually running cppwinrt. Just forgot to remove this as a part of the previous change.